src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -1993,6 +1994,17 @@ def load_arguments(self, _):
         c.argument("dns_zone_resource_ids", options_list=["--ids"], required=True)
         c.argument("attach_zones")
 
+    with self.argument_context('aks check-network outbound') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'],
+                   required=True, help='Name of the managed cluster.')
+        c.argument('resource_group', options_list=['--resource-group', '-g'],
+                   required=True, help='Name of the resource group.')
+        c.argument('node_name', options_list=['--node-name'],
```

Comment:
`--node-name` is default option format of parameter `node_name`, you can omit it if you don't plan to set another shorter option name
```suggestion
        c.argument('node_name',
```

Reply:
updated